misc/services/camel/de-metas-camel-externalsystems/core/readme.md

```diff
@@ -0,0 +1,24 @@
+
+### Metasfresh => Camel External System Dispatcher
+
+ - the entry point for any message is the `rabbitMQ` camel route consumer defined as: `rabbitmq:MF_TO_ExternalSystem`, with the following options:
+   - `durable=true`
+   - `autoDelete=false`
+   - `prefetchCount=1`
+     - *The maximum number of messages that the server will deliver* 
```

Comment:
i don't understand what that means..which server? deliver to whom? why did you choose refetchCount=1?